peripherals/syscfg/syscfg_f303_3x8.yaml

```diff
@@ -0,0 +1,276 @@
+SYSCFG:
+  CFGR1:
+    I2C3_FMP:
+      Standard: [0, "FM+ mode is controlled by I2C_Pxx_FMP bits only"]
+      # TODO: Different descripton to 37x / syscfg_f3.yaml
+      FMP: [1, "FM+ mode is enabled on all I2C3 pins selected through selection trhough IOPORT control registers AF selection bits"]
+    ENCODER_MODE:
+      NoRedirection: [0, "No redirection"]
+      # TODO: Naming?
+      MapTim2Tim15: [1, "TIM2 IC1 and TIM2 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim3Tim15: [2, "TIM3 IC1 and TIM3 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim4Tim15: [3, "TIM4 IC1 and TIM4 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively (STM32F303xB/C and STM32F358xC devices only)"]
+    DAC2_CH1_DMA_RMP:
+      # Note: In STM32F303x6/8 and STM32F328x8, this bit must be set
+      NoRemap: [0, "No remap"]
+      Remap: [1, "DAC2_CH1 DMA requests mapped on DMA1 channel 5"]
+    TIM7_DAC1_CH2_DMA_RMP:
+      # Note:    In STM32F303x6/8 and STM32F328x8, this bit must be set as there is no DMA2 in these products.
```

Comment:
Any way, to get this information into the generated code / documentation. Or is it up to the HAL implementer to get this right. I feel like, this bit is too hidden for this important(?) information. Though this is not really a topic we have to address, it's more on the side of STMicroelectronics.

Reply:
We don't have any way to provide this information really; it's up to the HAL writer or end user to read the RM and set this appropriately.